WalletWasabi/CoinJoin/Common/Models/RoundStateResponse.cs

```diff
@@ -64,7 +64,7 @@ public Money CalculateRequiredAmount(params Money[] queuedCoinAmounts)
 			}
 
 			return baseMinimum + FeePerInputs;
-			// return baseMinimum + (FeePerInputs * MaximumInputCountPerPeer);
+			//// return baseMinimum + (FeePerInputs * MaximumInputCountPerPeer);
```

Comment:
does this make sense?

```suggestion
			// return baseMinimum + (FeePerInputs * MaximumInputCountPerPeer);
```

Reply:
same as above, this is good, ACK.